### ISSUE TITLE
Local IP Address seen in WebRTC IP when using a proxy server on a different IP

### ISSUE THREAD

**Issue by henzycuong1** [seleniumbase/SeleniumBase#2217]
Code:
```
from seleniumbase import Driver
from seleniumbase.undetected import Chrome
driver:Chrome = Driver(
                browser='chrome',
                driver_version=None,
                uc=True,
                user_data_dir=user_dir,
                chromium_arg = f'--disable-gpu,{options}',
                proxy=proxy if proxy is not None else None
          )
```
Result:
![image](https://github.com/seleniumbase/SeleniumBase/assets/55269394/9d8abc5f-70cf-49e9-a3f8-0fd963cfec0b)

**Comment by mdmintz** [comment]
I see the issue. It'll be fixed as part of https://github.com/seleniumbase/SeleniumBase/issues/2218

Also, that's not a valid syntax format for SeleniumBase. See a list of valid formats here: [SeleniumBase/help_docs/syntax_formats.md](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/syntax_formats.md)

For the direct `driver` format, use something like this:

```python
from seleniumbase import Driver

driver = Driver(uc=True, proxy="HOST:PORT", user_data_dir=USER_DATA_DIR)
try:
    driver.get("https://browserleaks.com/webrtc")
    import pdb; pdb.set_trace()
finally:
    driver.quit()
```

**Comment by henzycuong1** [comment]
Sorry, I did't explain. It's just a part of ny code and "proxy" has correctly format "IP:PORT"

**Comment by mdmintz** [comment]
This was resolved in `4.20.8` - https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.20.8

Example:

```python
from seleniumbase import Driver

driver = Driver(uc=True, proxy="socks5://142.44.212.57:49006")
try:
    driver.get("https://browserleaks.com/webrtc")
    import pdb; pdb.set_trace()
finally:
    driver.quit()
```

<img width="1280" alt="Screenshot 2023-10-28 at 10 13 50 AM" src="https://github.com/seleniumbase/SeleniumBase/assets/6788579/ec2605d2-f045-4d59-a121-79eb5451c8f1">

**Comment by FranciscoPalomares** [comment]
<img width="438" alt="image" src="https://github.com/seleniumbase/SeleniumBase/assets/39648210/0bc8d43a-9754-4873-8bbd-fb8a2c0ecc42">
The ideal is that public ip appears as when using a chrome browser, safari.... without seleniumbase
